tests/system_tests/test_core/test_mode_disabled_full.py

```diff
@@ -24,10 +24,10 @@ def test_disabled_summary(user):
     run = wandb.init(mode="disabled")
     run.summary["cat"] = 2
     run.summary["nested"] = dict(level=3)
-    print(run.summary["cat"])
-    print(run.summary.cat)
+    _ = run.summary["cat"]
```

Comment:
any reason to store to `_`?

Reply:
ah just wanted to emphasize that it was a intentional access to the summary attribute

Reply:
Hmm maybe a more intentional assertion here might be?
```
assert "cat" in run.summary
assert 2 == run.summary.cat
```

similarly I think below is clearer on our intention for testing
```
assert "dog" not in run.summary
```
but for the dog assertion checking the error is raised seems fine as well to me

Reply:
i don't know if we change summary in disabled mode, let me verify

Reply:
ah we do it works, so i can update it......